packages/table/src/table.tsx

```diff
@@ -1157,9 +1157,9 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const columnIndices = this.grid.getColumnIndicesInRect(viewportRect, fillBodyWithGhostCells);
 
         const columnIndexStart = showFrozenColumnsOnly ? 0 : columnIndices.columnIndexStart;
-        const columnIndexEnd = showFrozenColumnsOnly ? numFrozenColumns : columnIndices.columnIndexEnd;
+        const columnIndexEnd = showFrozenColumnsOnly ? numFrozenColumns - 1 : columnIndices.columnIndexEnd;
```

Comment:
this (and the row below) are fencepost errors. `tableBodyCell` actually renders up to and including the end indices: https://github.com/palantir/blueprint/blob/783d8e26fd11c5a6d387d48152defe687865c74f/packages/table/src/tableBodyCells.tsx#L127

Reply:
You can remove changes in this file. These fencepost errors are already fixed in #1678.